module/MotionParser.cpp

```diff
@@ -111,7 +111,50 @@ vector<Motion*> MotionParser::createMotions(const char* commandFilePath, int tar
       motionList.push_back(xr);                                          // 動作リストに追加
     }
     */
-    else {  // 未定義のコマンドの場合
+    else if(command == COMMAND::IS) {                              // 交点内移動（直進）
+      InCrossStraight* is = new InCrossStraight(atof(params[1]),   // 目標距離
+                                                atof(params[2]));  // 目標速度 [mm/s]
+
+      motionList.push_back(is);                           // 動作リストに追加
+    } else if(command == COMMAND::IL) {                   // 交点内移動（左折）
+      InCrossLeft* il = new InCrossLeft(atof(params[1]),  // 目標距離
+                                        atof(params[2]),  // 距離指定直進の目標速度 [mm/s]
+                                        atof(params[3]),  // 角度指定回頭の目標速度 [mm/s]
+                                        atoi(params[4]));  // 目標回頭角度
```

Comment:
```suggestion
                                        atoi(params[3]),  // 目標回頭角度
                                        atof(params[4]));  // 角度指定回頭の目標速度 [mm/s]
```
上記のように、引数の順番逆がいいかな。
目標距離、直進の目標速度の順に指定するなら、それに従って目標回頭角度、回頭の目標速度の順が良いと思う。.cppと.hファイルの引数の順番も逆にするように。